index.js

```diff
@@ -302,13 +302,13 @@ module.exports = function (log, indexesPath) {
     }
   }
 
-  function safeReadUint32(buf) {
+  function safeReadUint32(buf, offset) {
```

Comment:
I'd rather put the default `0` in the function argument, with

```js
function safeReadUint32(buf, offset = 0) {
```

This is an ES6 feature supported "everywhere modern", even in IE11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

Reply:
Nice, I really those default arguments. They were really useful in c++.